.github/workflows/ci.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
       - name: Bring up mock endpoints for SSL verification
         run: docker-compose -f docker-compose.travis.yml up -d mock-endpoints.local
-
+       
```

Comment:
Code style nitpicking: can you remove this whitespace cfr. the `.editorconfig`?

Reply:
Already removed the newline change